SUMMARY.md

```diff
@@ -63,6 +63,7 @@
     * [How to start a process for a case](using-valtimo/document/start-case.md)
     * [Custom case columns](using-valtimo/document/custom-case-columns.md)
     * [Custom case headers](using-valtimo/document/custom-case-headers.md)
+    * [Search fields](using-valtimo/document/search-fields.md)
```

Comment:
Custom search fields? How to configure a search field?

Reply:
That information is not yet available because it hasn't been build yet

Reply:
My point is, this title is weird. We do this incorrectly for custom case columns and custom case headers as well, where we should really name the page what you'll learn by reading it. I think explaining how to use a search field is a little bit weird, but the second paragraph is called Configure search fields. This is all I would expect on this page, and the title of the page should cover that as well.